integration/travis/run_integration.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -ev
+
+export PROJECT_DIR=`pwd`
+$PROJECT_DIR/../travis/start_scheduler.sh
+sleep 60
```

Comment:
Would it be possible to poll the scheduler to check if it's ready (with a timeout) rather than `sleep`ing for 60 seconds? That way if it's ready after 30 seconds, we can start faster.